sdk/ai/azure-ai-client/samples/agents/user_functions.py

```diff
@@ -58,8 +59,8 @@ def send_email(recipient: str, subject: str, body: str) -> str:
 
 
 # Statically defined user functions for fast reference
-user_functions = {
-    "fetch_current_datetime": fetch_current_datetime,
-    "fetch_weather": fetch_weather,
-    "send_email": send_email,
-}
+user_functions: Set[Callable[[], Any]] = {
```

Comment:
Set[Callable[..., Any]] could be better, ... means that there can be multiple or none arguments

Reply:
changed